tests/notebooks/annotation.ipynb

```diff
@@ -45,19 +45,13 @@
    ],
    "source": [
     "import os\n",
-    "import json\n",
-    "with open('tests/notebooks/annotation.config.json') as f:\n",
-    "    config = json.load(f)\n",
-    "print(config)\n",
     "\n",
-    "n_epochs_all = config['n_epochs'] if 'n_epochs' in config else None\n",
-    "save_path = config['save_path'] if 'save_path' in config else 'data/'\n",
-    "n_samples_tsne = config['n_samples_tsne'] if 'n_samples_tsne' in config else None\n",
-    "n_samples_posterior_density = config['n_samples_posterior_density'] if 'n_samples_posterior_density' in config else None\n",
-    "train_size = config['train_size'] if 'train_size' in config else None\n",
-    "M_sampling_all = config['M_sampling'] if 'M_sampling' in config else None\n",
-    "M_permutation_all = config['M_permutation'] if 'M_permutation' in config else None\n",
-    "n_labelled_samples_per_class = config['n_labelled_samples_per_class'] if 'n_labelled_samples_per_class' in config else None"
+    "save_path = 'data/'\n",
+    "n_epochs_all = None\n",
+    "n_labelled_samples_per_class = None\n",
+    "M_sampling_all = None\n",
+    "M_permutation_all = None\n",
```

Comment:
I'm not sure I understand these. Why are they set to None? Don't we want to set them to the recommended values?

Reply:
I think the method in my last commit is better and clearer